packages/design-system/src/components/Button/Button.js

```diff
@@ -4,9 +4,27 @@ import classNames from 'classnames/bind';
 import styles from './style.scss';
 import getMergedClassNames from '../../utils/getMergedClassNames';
 import { Loader } from '../Loader';
+import { noticeAboutDeprecation } from '../../helpers/notice-about-deprecation';
 
 const cx = classNames.bind(styles);
 const acceptedSizes = ['large', 'compact'];
+const acceptedKinds = ['primary', 'secondary', 'destructive', 'text'];
+
+const getDeprecatedKind = (primary, destructive, secondary) => {
```

Comment:
if you have the mana (xD) for it you could move it to `helpers.ts`

Reply:
I thought about it, but it's not reusable at all, is it?

Reply:
yes, but the component file could be smaller and more readable. Nvm, ignore it. The file is not that big and this method is declared before component so it's all good :)